Flow.Launcher.Core/Updater.cs

```diff
@@ -22,23 +22,26 @@ namespace Flow.Launcher.Core
 {
     public class Updater
     {
-        public string GitHubRepository { get; }
+        private readonly IPublicAPI API;
```

Comment:
should the name be `api` rather than `API` as this is a filed rather than a property?